lib/matplotlib/text.py

```diff
@@ -1002,7 +1002,7 @@ def set_horizontalalignment(self, align):
 
         Parameters
         ----------
-        align : {'left', 'center', 'right'}
+        align : {'left', 'center', 'right'}, default: left
```

Comment:
There is now default value here as the `align` parameter is not optional (i.e. you can't call `set_horizontalalignment()` and expect that it sets the alignement to `left`)

Reply:
But if you don't call it, it is ``left``. How to convey that in the docs?

Reply:
The default values are (implicitly) documented in the [`__init__`](https://matplotlib.org/stable/api/text_api.html#module-matplotlib.text) method (class signature).

Further, matplotlib [follows](https://matplotlib.org/stable/devel/documenting_mpl.html#writing-docstrings) [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) conventions:

> When a parameter can only assume one of a fixed set of values, those values can be listed in braces, **with the default appearing first**:
> ```
>order : {'C', 'F', 'A'}
>    Description of `order`.
>```

So the explicit description of the default value is not required in this case, we'd just need to move the `baseline` entry for `verticalalignment` to the first position.

Reply:
> move the `baseline` entry for `verticalalignment` to the first position.

Done.